### PR TITLE
Update decision-graph and execution model docs to include 'initialize'

### PR DESCRIPTION
**Pull request by duncanmortimer**
Update to the documentation relevant to PR #223.

### CODE REVIEW

**Comment by ordnungswidrig** [comment]
+1 for updating docs. I think the "initialize" action should be added to https://github.com/clojure-liberator/liberator/blob/gh-pages/doc/actions.md, too.

**Comment by duncanmortimer** [comment]
Have updated existing docs to refer to ":initialize-context" and updated "actions.md".  Let me know if the documentation I've provided is sufficient.  Best, Duncan

**Comment by duncanmortimer** [comment]
Have also merged commits here.

File: doc/actions.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ layout: default
...
+```:initialize-context``` is the first action performed when a request
+is being handled --- its purpose is to allow additional values to be
+inserted into the context (in addition to the standard
+```:representation```, ```:request``` and ```:resource``` keys --- see
+the [execution model](execution-model.html) documentation) before the
+request is processed further.  Note that this action should not, in
+general, modify the state of the server --- it is intended solely to
+provide a convenient way to thread additional values through the
+execution flow.
+
```
Comment (ordnungswidrig) [inline doc/actions.md]: Please add annotation that this is unreleased, yet.

File: doc/execution-model.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ The context initially contains the following keys
 *  ````:request```` - the original ring request
 *  ````:resource```` - the map that defines the resource, including decision
    function etc.
+* Additional values can be included in the initial context using the
+  ````:initialize-context```` action.
```
Comment (ordnungswidrig) [inline doc/execution-model.md]: Please add annotation that this is unreleased, yet.

**Comment by duncanmortimer** [comment]
Have added annotations - hopefully what you meant?

**Comment by ordnungswidrig** [comment]
:+1:

**Comment by ordnungswidrig** [comment]
Thanks for the patch!

**Comment by duncanmortimer** [comment]
Great! Thanks!

**Pull request merged**